README.md

```diff
@@ -47,17 +47,12 @@ Examples of `Build` resource using the example strategies shipped with this oper
 
 ## Try it!
 
-- Install Tekton, optionally you could use
-[OpenShift Pipelines Community Operator][pipelinesoperator]
-
-- Install [operator-sdk][operatorsdk]
-
-- Create a project or namespace called **build-examples** by using `kubectl create namespace build-examples`
-
-- Execute `make local` to register [well-known build strategies](samples/buildstrategies) including **Kaniko**
-and start the operator.
-
-- Create a [Kaniko](samples/build/build_kaniko_cr.yaml) build
+* Get a [Kubernetes](https://kubernetes.io/) cluster and [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) set up to connect to your cluster, the supported version is v1.17.
```

Comment:
I think this can be leveraged with a table under a particular **Version** section or similar, something like:

|  Dependencies | Supported versions  | Current version   |
|---|---|---|
|  Tekton | v0.11.*  | v0.11.3  |
|  Kubernetes | 1.17.* , 1.16.* , 1.15.*  |  1.17 |
|   |   |   |   |   |

Just my 2 cents

Reply:
Yeah, I was too lazy yesterday evening to make it beautiful. I added a table now. I omitted the *Current version* column as I think that this can confuse. We should make sure that we test the latest patch level of all the supported versions through [Bleeding Edge Kubernetes CI #185](https://github.com/redhat-developer/build/issues/185)